importer/make_KMB_info.py

```diff
@@ -132,6 +133,8 @@ def load_mappings(self, update_mappings):
                 kmb_files_file, as_json=True)
             self.mappings['commonscat'] = common.open_and_read_file(
                 commonscat_file, as_json=True)
+            self.mappings['churches'] = common.open_and_read_file(
```

Comment:
put this on the same level as `self.mappings['countries']` since we want to use the same file even if we are doing an update_mappings, and the file is included in the repo so it will always exist.

Reply:
Done.